src/main/kotlin/org/malachite/estella/services/TaskStageService.kt

```diff
@@ -115,26 +117,70 @@ class TaskStageService(
         val newTaskResults = taskStage.tasksResult.filter { it.task.id != resultToSave.task.id }.plus(savedResult)
         taskStageRepository.save(taskStage.copy(tasksResult = newTaskResults))
     }
+
+    private fun isResultNew(it: TaskResult, resultToAdd: TaskService.ResultToAdd) =
```

Comment:
nie wiem, czy it jest najlepszą nazwą w metodzie

Reply:
sry, autogenerowana i nie zauważyłem xD